client/signup/main.jsx

```diff
@@ -565,6 +565,10 @@ class Signup extends Component {
 		// but it's not recommended outside of this, hence the name toStepper. See Automattic/growth-foundations#72 for more context.
 		if ( ! dependencies.toStepper ) {
 			debug( 'Tracking signup completion.', debugProps );
+			const isMapping = domainItem && isDomainMapping( domainItem );
+			const isTransfer = domainItem && isDomainTransfer( domainItem );
+			const isTransferOrMapping =
```

Comment:
I'd prefer not having a prop named `is...` contain a value

Reply:
You have a great point, will refactor 👍